src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -274,7 +272,11 @@ file that was distributed with this source code.
                 return true;
             });
 
-            {% if sonata_admin.field_description and sonata_admin.field_description.hasAssociationAdmin %}
+            // Automatically select the created record after closing the popup window
```

Comment:
Wait… this is not valid html, is it? Have you tested this?

Reply:
Everything has been tested with different use case (with & without create route) and is working.
What is not valid for you ?